### DISCUSSION TITLE
how can delete my folder on my repo

### DISCUSSION THREAD

**Discussion by Dasturchi-Yigit** [community/community#64350] [Repositories]
### Select Topic Area

Question

### Body

who can tell me to delete the spicific folder from my ropo. with desktop app or console ?

**Comment by ALISHERpy** [comment]
open it up with github,just it

**Comment by ALISHERpy** [comment]
thanks

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
